### PR TITLE
[Merged by Bors] -  add bors config after mirroring reset

### PR DESCRIPTION
**Pull request by TriplEight**

### CODE REVIEW

**Comment by TriplEight** [comment]
bors try

**Review by kirillt — APPROVED** [review]

**Comment by TriplEight** [comment]
added `bors` to waterfall repo. Hope everyone knows how to use it. 
It means that you shouldn't use the Big Green Button any more there.
Here's dashboard https://app.bors.tech/repositories/22637
And the manual https://bors.tech/documentation/

File: bors.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "ci/gitlab/%",
+  "continuous-integration/gitlab-build-and-test"
+]
+# The default is one hour.
+timeout_sec = 600
+
+# Do not allow pull requests with these labels to be merged.
+block_labels = [
+    "question"
+]
```
Comment (pepyakin) [inline bors.toml]: not sure if we actually need that.

**Review by pepyakin — APPROVED** [review]
I don't mind bors, so OK. the config looks good as far as my knowledge of bors goes (basically it is non-existent)

File: bors.toml (hunk 1 above)
Comment (TriplEight) [reply bors.toml]: It was just a harmless sample in case of you will want something like that. Can remove.

**Comment by TriplEight** [comment]
ah, should have removed the requirement of reviews from GH settings.

**Pull request closed without merging**